pkgs/tools/misc/gtkterm/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkg-config, gtk3, vte, libgudev }:
+stdenv.mkDerivation rec {
+  pname = "gtkterm";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "Jeija";
+    repo = "gtkterm";
+    rev = "${version}";
+    sha256 = "0s2cx8w1n8d37pl80gll5h6dyvbqrfcam8l4wmvnqqww9jml6577";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkg-config
```

When pressing the Log -> To file button, the app crashes as follows:
```
(gtkterm:124289): GLib-GIO-ERROR **: 20:43:38.383: No GSettings schemas are installed on the system
```
You can fix that by adding `wrapGAppsHook`:
```suggestion
    meson ninja pkg-config wrapGAppsHook
```

---

Interesting, on my system, `Log -> To` works well. Can it be because I run i3 WM and not, for example, Gnome? Anyway, I'll add the hook.

---

I'm actually on i3 as well, i3-gaps to be specific. Looking at the wrapper `wrapGAppsHook` produces it sets a few env vars, most notably here the line `XDG_DATA_DIRS='/nix/store/...-gtk+3-3.24.31/share/gsettings-schemas/gtk+3-3.24.31'$XDG_DATA_DIRS` which I think fixes the issue. My best guess is that you have those set in your environment already, perhaps GTK is enabled system-wide? Either way, now works perfectly for me as well :)